build.sh

```diff
@@ -139,54 +138,62 @@ while [[ $# -gt 0 ]]; do
 done
 
 
-# Build variables required by the docker compose command.
-
-BUILD_TASKS=()
-
-if [[ -n "$VERIFY_GENERATION" ]]; then
-  BUILD_TASKS+=('./scripts/verify_generation.sh')
-fi
-
-if [[ -n "$VERIFY_CODEBASE" ]]; then
-  BUILD_TASKS+=('./scripts/setup_database.sh')
-  BUILD_TASKS+=("./scripts/verify_codebase.sh")
-fi
-
-if [[ -n "$GENERATE_DOCS" ]]; then
-  BUILD_TASKS+=('./scripts/generate_docs.sh')
-fi
-
-BUILD_COMMAND="$(join_array ' && ' "${BUILD_TASKS[@]}")"
-echo "BUILD_COMMAND=${BUILD_COMMAND}"
-export BUILD_COMMAND
-
-FQL_COMMAND=${FQL_ARGS[*]}
-echo "FQL_COMMAND=${FQL_COMMAND}"
-export FQL_COMMAND
+# Build variables for fenv and docker commands.
 
 DOCKER_TAG="$(./scripts/docker_tag.sh)"
 echo "DOCKER_TAG=${DOCKER_TAG}"
 export DOCKER_TAG
 
-FDB_DOCKER_IMAGE="foundationdb/foundationdb:${FDB_VER:-6.2.30}"
-echo "FDB_DOCKER_IMAGE=${FDB_DOCKER_IMAGE}"
-export FDB_DOCKER_IMAGE
+# Set fenv environment variables
+export FENV_FDB_VER="${FDB_VER:-6.2.30}"
+export FENV_PROJECT_NAME="fql"
+
+# fenv.sh script location
+FENV_SCRIPT="./fenv/fenv.sh"
+
+# Common fenv flags
+FENV_FLAGS=()
+FENV_FLAGS+=(--docker ./docker/Dockerfile)
+FENV_FLAGS+=(--target fenv-builder)
```

Comment:
Combine the `FENV_SCRIPT` and `FENV_FLAGS` into a `FENV_CMD` variable which we can reuse below.